invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/License/LicenseModal.js

```diff
@@ -191,6 +191,8 @@ export class LicenseModal extends Component {
                     placeholder={i18next.t("License title")}
                     fieldPath="selectedLicense.title"
                     required
+                    // eslint-disable-next-line
```

Comment:
what was the eslint complaining about here?

Reply:
autoFocus is not good for accessibility